SPECIFICATIONS.md

```diff
@@ -1064,30 +1064,8 @@ For each block i from 0 to n:
 If all signatures are verified, extract pk_n+1 from the last block and
 sig from the proof field, and check `verify(pk_n+1, sig_n+1, data_n+alg_n+1+pk_n+1+sig_n)`
 
-### Blocks
 
-A block is defined as follows in the schema file:
-
-```proto
-message Block {
-  repeated string symbols = 1;
-  optional string context = 2;
-  optional uint32 version = 3;
-  repeated FactV2 facts_v2 = 4;
-  repeated RuleV2 rules_v2 = 5;
-  repeated CheckV2 checks_v2 = 6;
-  repeated Scope scope = 7;
-  repeated PublicKey publicKeys = 8;
-}
-```
-
-The block index is incremented for each new block. The Block 0
-is the authority block.
-
-Each block can provide facts either from its facts list, or generate
-them with its rules list.
-
-### Symbol table
+## Symbol table
```

Comment:
the symbol table section is moving up a level. Mayb it should move under the format section though?

Reply:
maybe we should rename it to "string interning", since it’s not just a serialization matter